docs/core/versions/selection.md

```diff
@@ -53,6 +53,10 @@ The process for selecting an SDK version is:
 
 For more information about SDK version selection, see the [Matching rules](../tools/global-json.md#matching-rules) and [rollForward](../tools/global-json.md#rollforward) sections of the [global.json overview](../tools/global-json.md) article.
 
+### Updating the SDK version
+
+It is important to update to the latest version of the SDK regularly to adopt the latest features, performance improvements, and bug fixes. To easily check for updates to the SDK, use the `dotnet sdk check` [command](../tools/dotnet-sdk-check.md). Additionally, if you specify a specific version using *global.json*, consider a tool such as Dependabot to automatically update pinned the SDK as new versions become available.
```

Comment:
Can you fix the last part of this paragraph?  I think this is what you meant:

```suggestion
It is important to update to the latest version of the SDK regularly to adopt the latest features, performance improvements, and bug fixes. To easily check for updates to the SDK, use the `dotnet sdk check` [command](../tools/dotnet-sdk-check.md). Additionally, if you select a specific version using *global.json*, consider a tool such as Dependabot to automatically update the pinned SDK version as new versions become available.
```

Reply:
Oof, yes that was bad. I accepted your suggestion. Thank you for reviewing!